security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/modules/network/variables.tf

```diff
@@ -0,0 +1,57 @@
+###############################################################################
+# Copyright (c) 2022, 2023, Oracle and/or its affiliates.  All rights reserved.
+# This software is dual-licensed to you under the Universal Permissive License
+# (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+###############################################################################
+#
+# Author: Fabrizio Zarri
+#
+################################################################################
+
+variable "deployment_name" {
+  description = "Deployment Name"
+}
+variable "region" {
+  description = "OCI Region"
+}
+
+variable "purpose" {
+}
+
+variable "compartment_ocid" {
+  description = "the OCID of the compartment where the environment will be created."
+}
+
+variable "VCN-CIDR" {
+  description = "The CIDR block of VCN"
+}
+
+variable "subnet-CIDR" {
```

Comment:
See comment on line 25.

Reply:
Resolved, I prefer to offer this default configuration.